### PR TITLE
ddd 64 endpoint: return all orders 

### PR DESCRIPTION
**Pull request by egisxxegis**
returns orders of all users.
address for GET: `http://localhost:8080/delivery-service/api/v1/orders/all`
Yet to do: make it accessible only for admin

### CODE REVIEW

File: src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java [hunk 1]
```diff
@@ -60,4 +64,37 @@ public Response listAction() {
                 .ok(new ListOrderDTO(GetOrderDTO.createMany(orders)))
                 .build();
     }
+
+    @GET
+    @Path("/all")
```
Comment (giplonis) [inline src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java]: This `all` is redundant. Generally `orders` already means "all" orders

File: src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java [hunk 2]
```diff
@@ -60,4 +64,37 @@ public Response listAction() {
...
+    @Authorized //todo make it admin-only
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Fetch list of all orders",
+            tags = { "Order" },
+            responses = {
+                    @ApiResponse(
```
Comment (giplonis) [inline src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java]: Wrong indentation - let's keep things consistent 🙂

File: src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java [hunk 3]
```diff
@@ -60,4 +64,37 @@ public Response listAction() {
...
+                            responseCode = "200",
+                            content = @Content(schema = @Schema(implementation = ListOrderDTO.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "500",
+                            content = @Content(schema = @Schema(implementation = ExceptionDTO.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "400",
```
Comment (giplonis) [inline src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java]: Status code 400 should go before 500 - let's keep things consistent

File: src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java [hunk 4]
```diff
@@ -60,4 +64,37 @@ public Response listAction() {
...
+                            content = @Content(schema = @Schema(implementation = ExceptionDTO.class))
+                    )
+            }
+    )
+    public Response listAllAction() {
```
Comment (giplonis) [inline src/main/java/lt/vu/web/api/v1/controller/order/ListOrderController.java]: I think whole admin api should be placed into separate package to not confuse things together. Let's say `lt.vu.web.api.v1.admin.{same as regular api stuff}`

File: src/main/java/lt/vu/web/api/v1/admin/controller/order/ListOrderController.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+@Path("/admin/orders")
```
Comment (egisxxegis) [inline src/main/java/lt/vu/web/api/v1/admin/controller/order/ListOrderController.java:24]: new Path

**Review by giplonis — APPROVED** [review]

**Pull request merged**